Libraries/Alert/Alert.js

```diff
@@ -27,6 +27,7 @@ export type Buttons = Array<{
 
 type Options = {
   cancelable?: ?boolean,
+  userInterfaceStyle?: string,
```

Comment:
Idea: Can we specify the exact wordings to prevent setting unexpected values here?

Something like below?

```
'unspecified' | 'light' | 'dark'
```

And/Or we can use RCT_ENUM_CONVERTER for RCTAlertViewStyle as a reference. It has a nice conversion 👍

https://github.com/facebook/react-native/blob/main/React/CoreModules/RCTAlertManager.mm#L22

Reply:
> Idea: Can we specify the exact wordings to prevent setting unexpected values here?
> 
> Something like below?
> 
> ```
> 'unspecified' | 'light' | 'dark'
> ```
> 
> And/Or we can use RCT_ENUM_CONVERTER for RCTAlertViewStyle as a reference. It has a nice conversion 👍
> 
> https://github.com/facebook/react-native/blob/main/React/CoreModules/RCTAlertManager.mm#L22

Thank you for your suggestion, I have modified as suggested。